src/annotator/test/annotation-sync-test.js

```diff
@@ -45,96 +45,115 @@ describe('AnnotationSync', function() {
     sandbox.restore();
   });
 
-  describe('#constructor', function() {
-    context('when "deleteAnnotation" is published', function() {
-      it('calls emit("annotationDeleted")', function() {
-        var ann = {id: 1, $tag: 'tag1'};
-        var eventStub = sinon.stub();
-        options.on('annotationDeleted', eventStub);
-        createAnnotationSync();
-
-        publish('deleteAnnotation', {msg: ann}, function() {});
+  function addTests() {
```

Comment:
The problem here is that I want to run all of `AnnotationSync`'s tests in two contexts: with and without a broken `Function.bind` in the parent page. Maybe not _all_ the tests really need to be run with the broken `Function.bind`, but a few of them should be, and this avoids duplicating them.

It does add some "cleverness" to the test code though.

An alternative would be to just add _one_ test for the broken Function.bind context: that the constructor doesn't crash. Since the broken Function.bind that the tests are using is one that crashes when called, just this one test would prove that `AnnotationSync` doesn't use the document's Function.bind